pom.xml

```diff
@@ -895,7 +895,7 @@
 
     <properties>
         <!-- Carbon kernel version-->
-        <carbon.kernel.version>4.6.1-alpha</carbon.kernel.version>
+        <carbon.kernel.version>4.6.1-beta</carbon.kernel.version>
```

Comment:
Need to change carbon.commons version once the PR [1] merged 
[1] https://github.com/wso2/carbon-commons/pull/423